data/datasets_test.js

```diff
@@ -14,9 +14,9 @@ export default {
       { type: 'doughnut', group_by: 'sex' },
     ],
     filter_configs: [
-      { name: 'year' },
-      { name: 'race' },
-      { name: 'sex' },
+      { type: 'checkbox-group', name: 'year' },
+      { type: 'checkbox-group', name: 'race' },
+      { type: 'checkbox-group', name: 'sex' },
```

Comment:
It's surprising to me that `chart_configs` and `filter_configs`, which seem like configs that are not specific to one chart, are duplicated in each chart. How do we define the data schema? Is this something we can change, or does it follow some existing standard?

Reply:
Hey @nholden , this is something I wasn't taking into consideration when I updated the explore page, so I'm glad you are working on this and caught onto it. I should have taken a bit more care when I created the test JSON files to make them more production like. If you want to update them so that you can properly integrate the ability for the schema to define filter type, please go for it.

Here are a couple of things for you to look at that may help:

1. [The current production Explore page](http://texasjusticeinitiative.org/data/) - We are basically recreating this page as is. So you can see here what we envision for the filter options and what the actual final data looks like.
2. [The production data schema](https://github.com/texas-justice-initiative/website-nextjs/blob/master/data/datasets.js) - You can see here the 3 data sets we will be loading with their corresponding schema. The URL for the current production JSON is in there as well, so you can see what have been working with. Note that we are updating the format of the JSON to simplify the charting process, which is why I created the small test JSON files.

Reply:
Thanks for the links and context, @LayaTaal! In https://github.com/texas-justice-initiative/website-nextjs/pull/61/commits/e05c62bdaa59dd8a9d7ad33f7bb530ba8b482683, I updated the PR to assume that if no `type` is specified for a filter, we'll assume we want to render it as checkboxes, like we do in the production data schema.

In a separate PR or PRs, I'll start to work on creating an autocomplete component that we can render when the `type` for a filter is specified as `autocomplete`.